app/controllers/FaciaToolController.scala

```diff
@@ -31,39 +32,56 @@ class FaciaToolController(
                            val mediaServiceClient: MediaServiceClient,
                            val deps: BaseFaciaControllerComponents
                          )(implicit ec: ExecutionContext)
-  extends BaseFaciaController(deps) with BreakingNewsEditCollectionsCheck with Logging {
+  extends BaseFaciaController(deps) with BreakingNewsEditCollectionsCheck with ModifyCollectionsPermissionsCheck with Logging {
```

Comment:
See comment below (maybe we can use an `ActionBuilder` here). It would reduce the size of the diff to hopefully just the bits that affect permission checks.